src/web/server/document.tsx

```diff
@@ -102,16 +169,12 @@ export const document = ({ data }: Props) => {
 	 * *before* the high priority scripts, although this is very
 	 * unlikely.
 	 */
-	const lowPriorityScriptTags = generateScriptTags(
-		[
-			{ src: 'ophan.js', module: true },
-			{ src: 'lotame.js', module: true },
-			{ src: 'atomIframe.js', module: true },
-			{ src: 'embedIframe.js', module: true },
-			{ src: 'newsletterEmbedIframe.js', module: true },
-		],
-		'async',
-	);
+	const lowPriorityScriptTags = generateScriptTags([
+		{ src: 'lotame.js', module: true },
+		{ src: 'atomIframe.js', module: true },
+		{ src: 'embedIframe.js', module: true },
+		{ src: 'newsletterEmbedIframe.js', module: true },
+	]);
```

Comment:
It was removed from `lowPriorityScriptTags` in my previous PR because we had a while ago added it to high priority but not removed it from lowPriority (Commercial bundle currently relies on Ophan being loaded)